assets/js/modules/ads/components/settings/SettingsView.js

```diff
@@ -57,9 +59,37 @@ export default function SettingsView() {
 
 	const isPaxView = paxEnabled && ( paxConversionID || extCustomerID );
 
+	const isConversionTrackingEnabled = useSelect( ( select ) =>
+		iceEnabled ? select( CORE_SITE ).isConversionTrackingEnabled() : false
+	);
```

Comment:
It might be clearer to use conditional check and return early, instead of ternary operator, also to align with the rest of the codebase where we usually use it that way.

```js
const isConversionTrackingEnabled = useSelect( ( select ) =>
	if ( ! iceEnabled ) {
	     return false;
	} 
	
	return select( CORE_SITE ).isConversionTrackingEnabled();
);
```